examples/networking/hub-and-spoke-vpn/README.md

```diff
@@ -1,8 +1,15 @@
 # Hub and Spoke via VPN
 
-This example creates a simple **Hub and Spoke VPN** setup, where the VPC network connects satellite locations (spokes) through a single intermediary location (hub) via [IPsec VPN](https://cloud.google.com/vpn/docs/concepts/overview), optionally providing full-mesh networking via [custom route advertisements](https://cloud.google.com/router/docs/how-to/advertising-overview).
+This example creates a simple **Hub and Spoke VPN** setup, where the VPC network connects satellite locations (spokes) through a single intermediary location (hub) via [IPsec HA VPN](https://cloud.google.com/network-connectivity/docs/vpn/concepts/overview#ha-vpn).
 
-The example has been purposefully kept simple to show how to use and wire the VPC and VPN modules together, and so that it can be used as a basis for more complex scenarios. This is the high level diagram:
+A few additional features are also shown:
+
+- [custom BGP advertisements](https://cloud.google.com/router/docs/how-to/advertising-overview) to implement optional full mesh
+- [VPC Global Routing](https://cloud.google.com/network-connectivity/docs/router/how-to/configuring-routing-mode) to use a regional set of VPN gateways from subnets in different regions (used here for illustrative purposes of course)
```

Comment:
`regional set of VPN gateways from subnets in different regions (used here for illustrative purposes of course)` ... as next hops? Also, VPN gateways don't actually consume subnets. I'd probably remove this, especially b/c that's the default VPC behavior.

Reply:
...and I just got what you meant. subnets in different regions consume vpn gws. Took me a while to decode, maybe rephrase? Or maybe I need another coffee.

Reply:
yep, will rephrase :)